### PR TITLE
Instances in exercises

### PR DESCRIPTION
**Pull request by Droogans**
- [X] Dev LGTM
 - [x] Dev LGTM

### CODE REVIEW

**Comment by Droogans** [comment]
Oh hold until #1121 is merged (it's in this PR, it'll need to be rebased out).

**Comment by Droogans** [comment]
This needs to have the readme for rx-page-objects updated.

File: grunt-tasks/component-template/root/component.exercise.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var {%= name %} = require('./{%= name %}.page').{%= name %};
    {%= name %} exercises.
    @exports encore.exercise.{%= name %}
    @param {Object} [options=] - Test options. Used to build valid tests.
-   @param {string} [options.cssSelector=] - Fallback selector string to initialize widget with.
+   @param {{%= name %}} [options.instance={%= name %}.main] - Component to exercise.
    @example
    ```js
    describe('default exercises', encore.exercise.{%= name %}({
```
Comment (megalord) [inline grunt-tasks/component-template/root/component.exercise.js]: I can't comment on the line below, but don't forget to change that too.

Comment (Droogans) [inline grunt-tasks/component-template/root/component.exercise.js]: :+1: I'll ack for `cssSelector` I'm sure there's more.

**Comment by parlarjb** [comment]
LGTM

File: grunt-tasks/component-template/root/component.exercise.js (hunk 1 above)
Comment (CITguy) [inline grunt-tasks/component-template/root/component.exercise.js]: @Droogans, has this been addressed?

**Comment by Droogans** [comment]
@parlarjb @CITguy is there anything holding this back?

If new components get added, I have to do a lot of re-work to get this mergable.

File: grunt-tasks/component-template/root/component.exercise.js (hunk 1 above)
Comment (Droogans) [inline grunt-tasks/component-template/root/component.exercise.js]: Yes.

**Comment by megalord** [comment]
LGTM

**Pull request merged**